py/selenium/webdriver/common/actions/wheel_actions.py

```diff
@@ -29,6 +29,6 @@ def pause(self, duration=0):
         self.source.create_pause(duration)
         return self
 
-    def scroll(self, x, y, delta_x, delta_y, duration, origin):
+    def scroll(self, x=0, y=0, delta_x=0, delta_y=0, duration=0, origin="viewport"):
```

Comment:
be nice to type these; `int/float?` `float` if either is acceptible

Reply:
I had types and then removed them. I want to allow comparing apples to apples for this PR. We're going to do a full update on typing in the future.